src/orders/IOrderProcessor.sol

```diff
@@ -105,6 +105,11 @@ interface IOrderProcessor {
     /// @return Returns the total amount of the specific token held in escrow for the given user
     function escrowedBalanceOf(address token, address user) external view returns (uint256);
 
+    /// @notice This function retrieves the number of decimal places configured for a given token
+    /// @param token The address of the token for which the number of decimal places is fetched
+    /// @return Returns the number of decimal places set for the specified token
+    function orderDecimals(address token) external view returns (uint256);
```

Comment:
rename maxOrderDecimals. It is the maximum assetTokenQuantity decimal places supported for a specific assetToken